src/match_regex.c

```diff
@@ -31,6 +31,11 @@
  * Perl-compatible regular expressions.
  */
 
+#ifdef WIN32
+#include <gnulib_config.h>
+#include <config.h>
+#endif /* WIN32 */
```

Comment:
You usually indent the "include" in such blocks and omit the comment after #endif. Some consistency?